scripts/shared/backport.sh

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env bash
+
```

Comment:
This should mention where the original is (https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md).